src/applications/facility-locator/containers/FacilityLocatorApp.jsx

```diff
@@ -44,9 +44,9 @@ class FacilityLocatorApp extends React.Component {
       <Link to={this.buildSearchString()} key="facility-locator">Find Facilities & Services</Link>,
     ];
 
-    if (location.pathname.match(/facility\/[a-z]+_\d/) && selectedResult) {
+    if (location.pathname.match(/facility\/[a-z]{1,10}_[a-zA-Z0-9]{1,10}/) && selectedResult) {
       crumbs.push(<Link to={`/${selectedResult.id}`} key={selectedResult.id}>Facility Details</Link>);
-    } else if (ccLocatorEnabled() && location.pathname.match(/provider\/[a-z]+_\d/) && selectedResult) {
+    } else if (ccLocatorEnabled() && location.pathname.match(/provider\/[a-z]{1,10}_[a-zA-Z0-9]{1,10}/) && selectedResult) {
```

Comment:
I'm thinking that is the logic that uses the regex could be pulled out into a helper function it would make it super easy to test the crap out of the regex to make sure it worked as intended, didn't throw false-positives, etc.

Reply:
What is being determined by this regex? Is it represented more explicitly on the state or in the facilities response?